lib/api/response_models/app_response.dart

```diff
@@ -3,19 +3,19 @@ import 'package:client_common/utils/color_parser.dart';
 import 'package:flutter/material.dart';
 
 class AppResponse extends ApiResponse {
-  int id;
-  String name;
-  String serviceName;
-  IconData icon;
-  Color color;
+  int? id;
+  String? name;
+  String? serviceName;
+  IconData? icon;
+  Color? color;
```

Comment:
Some of those elements are needed:
```suggestion
  String name;
  String serviceName;
  IconData? icon;
  Color color;
```